receiver/dockerstatsreceiver/receiver.go

```diff
@@ -135,7 +135,10 @@ func (r *receiver) recordContainerStats(now pcommon.Timestamp, containerStats *d
 		metadata.WithContainerHostname(container.Config.Hostname),
 		metadata.WithContainerID(container.ID),
 		metadata.WithContainerImageName(container.Config.Image),
-		metadata.WithContainerName(strings.TrimPrefix(container.Name, "/")))
+		metadata.WithContainerName(strings.TrimPrefix(container.Name, "/")),
+		metadata.WithContainerImageID(container.Image),
+		metadata.WithContainerCommand(strings.Join(container.Config.Cmd, ",")),
```

Comment:
Take a look here: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/process.md the `process.command_args` uses a []string representation. Would it make sense to follow that approach here too?

Reply:
It would make sense since it's also what docker's API is returning. However, I was leveraging the `WithContainerCommand` func autogenerated with `mdatagen` and I think I cannot set the resource attribute in the `metadata.yaml` as a []string. Maybe I'm missing something and you can point me in the right direction, thanks!

Reply:
I updated it from `container.command` to `container.command_line` to be reported as a string and to keep consistency with the conventions as defined here: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/process.md#process (`process.command_line`). WDYT?

You can check the update in this commit f2b30c3e50f77cb1eb59b42426da5f4c0e928ae7.